src/main/java/com/codegile/univ/mihai/week1/day1/Project2.java

```diff
@@ -0,0 +1,46 @@
+package com.codegile.univ.mihai.week1.day1;
+import java.io.*;
+
+
+public class Project2 {
+
+    public static void main(String args[]){
+        FileInputStream fis;
+        InputStreamReader isr;
+        BufferedReader br;
+        int count = 0;
+        int c = 0;
+        try{
+            fis = new FileInputStream("java/com/codegile/univ/mihai/week1/day1/name.txt");
+            isr = new InputStreamReader(fis);
+            br = new BufferedReader(isr);
+
+            String dataline = br.readLine();
+            while(dataline != null){
+                System.out.println(dataline);
+                if((int)dataline.charAt(0) < 91 && (int)dataline.charAt(0) > 64){
+                    for(int i= 1; i<dataline.length();i++){
+                        if((int)dataline.charAt(i) > 96 && (int)dataline.charAt(i) < 123){
```

Comment:
try to think of a solution using RegEx
http://www.vogella.com/tutorials/JavaRegularExpressions/article.html